test/test-fs.c

```diff
@@ -309,6 +309,9 @@ static void chown_root_cb(uv_fs_t* req) {
 #   if defined(__CYGWIN__)
     /* On Cygwin, uid 0 is invalid (no root). */
     ASSERT(req->result == UV_EINVAL);
+#   elif defined(__PASE__)
+    /* On IBMi PASE, the permission is different. */
```

Comment:
Different how? Can you expand the comment?

Reply:
> Different how? Can you expand the comment?

updated below comments --
```
    /* On IBMi PASE, there is no root user. uid 0 is user qsecofr.
     * User may grant qsecofr's privileges, including changing 
     * the file's ownership to uid 0.
     */
```